src/Groups/GAPGroups.jl

```diff
@@ -914,7 +917,10 @@ Base.:^(H::GAPGroup, y::GAPGroupElem) = conjugate_group(H, y)
 """
```

Comment:
```suggestion
@doc raw"""
```
If you want to use `$` as you did.

Reply:
If the LaTeX is better I'm open to it, but I think it maybe didn't fit well mixed in with everything else; I ended up taking the LaTeX out (I do know how to typeset it now in the normal way though, and good to know this lets the `$` work).